lib/react/renderer.rb

```diff
@@ -40,12 +41,14 @@ def context
       @context ||= ExecJS.compile(self.class.combined_js)
     end
 
-    def render(component, args={})
+    def render(component, args={}, replay_console=false)
       react_props = React::Renderer.react_props(args)
       jscode = <<-JS
-        function() {
-          return React.renderToString(React.createElement(#{component}, #{react_props}));
-        }()
+        (function () {
+          var result = React.renderToString(React.createElement(#{component}, #{react_props}));
+          #{replay_console ? React::Console.replay_as_script_js : ''}
```

Comment:
I think this is addressed by the `JSON.stringify` https://github.com/reactjs/react-rails/pull/196#discussion-diff-25880836R22